api/src/main/resources/workbench.yaml

```diff
@@ -650,6 +650,10 @@ definitions:
       selectable:
         description: specifies if user can search with
         type: boolean
+      conceptId:
```

Comment:
From the docs, it's unclear how `code` and `conceptId` differ. Could the descriptions make this clear?

Reply:
I will update the descriptions to try a make this clearer.